src/main/flight/imu.h

```diff
@@ -17,6 +17,11 @@
 
 #pragma once
 
+#include "common/axis.h"
+#include "common/maths.h"
+
+#include "sensors/acceleration.h"
```

Comment:
`maths.h` and `acceleration.h` no longer required because of forward decalarations.